syntax = "proto3";

package mock;

message MockRequest {
}

message MockResponse {
}

service MockService {
    rpc MockRPC (MockRequest) returns (MockResponse);
}
